yt/visualization/plot_container.py

```diff
@@ -302,7 +302,9 @@ def set_minorticks(self, field, state):
         """
         if isinstance(state, str):
             issue_deprecation_warning(
-                "Deprecated api, use bools for *state*.", removal="4.1.0"
+                "Deprecated api, use bools for *state*.",
+                deprecation_id="plot_container:bools_for_state",
+                removal="4.1.0",
```

Comment:
```suggestion
                deprecation_id="plot_container:bools_for_state",
                since="4.0.0",
                removal="4.1.0",
```